vasodecanto-app/vasodecanto-app/Features/Login/Login.swift

```diff
@@ -7,7 +7,6 @@ struct LoginView: View {
                 .imageScale(.large)
                 .foregroundStyle(.tint)
             Text("Hello, world!")
-        }
-        .padding()
+        }.padding()
```

Comment:
para melhor legebilidade, é sempre bom manter o modifiers abaixo do final do bloco de código (abaixo das chaves).

```suggestion
        }
        .padding()
```